### PR TITLE
General clean up

### PR DESCRIPTION
**Pull request by loganmorganml1**
--Fixed format issues with form in the header on homepage 
--deleted extra div closing tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,73 +14,67 @@
...
+                       <span class="hamburger"></span>
+                   </div>
+               </nav>
+           </div>
+       </section>
+       <section class="hero">
+           <div class="hero-img-box">
+               <img class="hero-img" src="./assets/img/heros/hero-home.jpg" alt="photo of man looking at stars">
+           </div>
+           <div class="hero-content">
```
Comment (techeduk8r) [inline index.html:43]: Was the code copied/pasted from another version? The adjustment for excluding the hero-h1 from screenreading are gone. Easy enough to adjust once this is merged though.

**Comment by loganmorganml1** [comment]
Oh yes sorry, I took the header from a previous merge.
________________________________

**Comment by techeduk8r** [comment]
No worries. Just confirming. Let's merge and I make that small adjustment. It's commenting out the h1 and the rest should be fine because the CSS is already there. Unless you want to go ahead and comment out the hero-h1 with this merge.

**Comment by loganmorganml1** [comment]
Lets do that -- its just commenting out the CSS? You can also go in and make the update as well if you'd like, its in generalCleanUp​
________________________________

**Comment by techeduk8r** [comment]
Actually, in live server it looks fine. I'll approve and merge.

**Review by techeduk8r — APPROVED** [review]
Issues seems resloved.

**Pull request merged**